cypress/e2e/hub/namespaces.cy.ts

```diff
@@ -132,4 +132,27 @@ describe('Namespaces', () => {
 
     cy.deleteNamespace(namespaceName);
   });
+
+  it('user can bulk dekete namespaces', () => {
+    let namespaceName: string;
+
+    for (let i = 1; i <= 5; i++) {
+      namespaceName = `namespace_bulk_delete_${i}_${randomString(5, undefined, {
```

Comment:
@acruzgon This can filter 5 existing namespaces and delete them. Ideally, we should create a unique signature for the created namespaces, filter by that signature, bulk delete all of those, and then filter again to ensure that no namespace with the unique signature remains. This approach is just to be less error-prone regarding which namespaces would be deleted. See the remotes and remote-registries tests for an idea